### PR TITLE
feat(indexer): Hook up api to cli app

### PR DESCRIPTION
**Pull request by roninjin10**
This is a really old pr that never landed

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #6768** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6768" target="_blank">Graphite</a> 
    * **PR #6726** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6726" target="_blank">Graphite</a>  👈
      * **PR #6799** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6799" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6726?utm_source=stack-comment).

File: indexer/cli/cli.go [hunk 1]
```diff
@@ -63,16 +64,21 @@ func runApi(ctx *cli.Context) error {
...
+
+	if err != nil {
+		logger.Crit("Failed to connect to database", "err", err)
+	}
+
+	server := api.NewApi(db.BridgeTransfers, logger)
 
-	// finish me
-	return err
+	return server.Listen(strconv.Itoa(conf.API.Port))
```
Comment (hamdiallam) [inline indexer/cli/cli.go]: LGTM! But similar to `runIndexer`, we probably want catch interrupts/panics and gracefully handle them

**Review by hamdiallam — APPROVED** [review]

File: indexer/cli/cli.go (hunk 1 above)
Comment (hamdiallam) [reply indexer/cli/cli.go]: feel free to ignore this for now and handle this later

File: indexer/cli/cli.go [hunk 2]
```diff
@@ -101,7 +107,7 @@ func (c *Cli) Run(args []string) error {
 }
 
 func NewCli(GitVersion string, GitCommit string, GitDate string) *Cli {
-	flags := append([]cli.Flag{ConfigFlag}, log.CLIFlags("INDEXER")...)
+	flags := []cli.Flag{ConfigFlag, IndexerRefreshFlag}
```
Comment (roninjin10) [inline indexer/cli/cli.go]: I think another pr nukes IndexerRefresh flag will handle it based on which merges first

File: indexer/cli/cli.go (hunk 1 above)
Comment (roninjin10) [reply indexer/cli/cli.go]:
```suggestion
        // FIXME similar to runIndexer, we probably want catch interrupts/panics and gracefully handle them
	return server.Listen(strconv.Itoa(conf.API.Port))
```

**Pull request merged**